tools/expotools/src/Npm.ts

```diff
@@ -0,0 +1,92 @@
+import { spawnAsync, spawnJSONCommandAsync } from './Utils';
+
+export const EXPO_DEVELOPERS_TEAM_NAME = 'expo:developers';
+
+export type PackageViewType = null | {
+  name: string;
+  version: string;
+  'dist-tags': { [tag: string]: string };
+  versions: string[];
+  time: {
+    created: string;
+    modified: string;
+    [time: string]: string;
+  };
+  maintainers: string[];
+  description: string;
+  author: string;
+  gitHead: string;
+  // and more but these are the basic ones, we shouldn't need more.
+  [key: string]: any;
```

Comment:
```suggestion
  [key: string]: unknown; 
```
Always prefer `unknown` over `any` as calling `unknows` unintentionally will always trigger ts error while calling `any` is just ok 😉